src/main/java/org/htwkvisu/controller/ApplicationController.java

```diff
@@ -22,6 +24,20 @@
 public class ApplicationController implements Initializable {
 
     @FXML
+    private NumericTextField pixelDensityTextField;
+
+    private static final int DEFAULT_PIXEL_DENSITY = 30;
+    private static final int MAX_NUMERIC_FIELD_LENGTH = 6;
+
+    @FXML
+    private NumericTextField minScoringTextField;
+    private static final int DEFAULT_MIN_SCORING_VALUE = 0;
```

Comment:
könnte extrahiert werden in einer externen Klasse die sich um das NumericTextField kümmert und die DefaultValues hält. Damit würde diese 500 Zeilen Klasse etwas kleiner werden, was toll wäre.